eng/pipelines/vmr-sync-internal.yml

```diff
@@ -17,7 +17,7 @@ parameters:
 - name: vmrBranch
   displayName: dotnet-dotnet branch to push to
   type: string
-  default: ' '
```

Comment:
I think we want `' '` as we found out that `''` means it starts to be required which is not what we want?

Reply:
yes, my bad, I didn't see the space in the field and suddenly my branch name was starting with a space

Reply:
Yeah, that's not great. We could trim it in bash before using. I am doing the same here actually: https://github.com/dotnet/source-build/blob/820030c306ef2feed4dabc57ed905044e8efb4ac/eng/templates/steps/get-build-info.yml#L50